plugins/nf-validation/src/resources/META-INF/extensions.idx

```diff
@@ -1 +1,2 @@
 nextflow.validation.SchemaValidator
+nextflow.validation.SamplesheetConverter
```

Comment:
As the function is included in `main/nextflow/validation/SchemaValidator.groovy` I think this is not needed anymore.

Reply:
That's right! Forgot to remove it :)